### PR TITLE
Clarify Hamiltonian problem

### PR DESCRIPTION
**Pull request by darabos**
Gábor was a bit confused by what _"with 12 connections"_ applied to. I think it's clearer this way.

### CODE REVIEW

File: hiring/galactic.html [hunk 1]
```diff
@@ -105,8 +105,8 @@ <h1 class="text-center">Galactic <span class="highlight">&#9672;</span> Recursio
     <exercise id="hamiltonian"
     before="
 var sensors = ['radio', 'microwave', 'infrared', 'visible', 'ultraviolet', 'x-ray', 'gamma', 'neutrino', 'graviton', 'tachyon', 'quantum', 'intuiton'];
-// Returns the number of different circuits that can be built from the list of sensors with 12 connections.
-// The circuit (an undirected graph) must contain a Hamiltonian cycle due to safety regulations.
+// Returns the number of different circuits that can be built from the list of sensors.
+// The circuit (an undirected graph with 12 vertices and 12 edges) must contain a Hamiltonian cycle due to safety regulations.
```
Comment (xandrew-lynx) [inline hiring/galactic.html]: But now this makes the 12 vertices/edges be part of the "definition" of circuit. Which is quite a weird definition for a circuit and thus confusing. Maybe we just need a better explanation of what a sensot circuit is, using words like wires and sensors and then say something like we have 12 wires.

Comment (darabos) [inline hiring/galactic.html]: > But now this makes the 12 vertices/edges be part of the "definition" of circuit. Which is quite a weird definition for a circuit and thus confusing. Maybe we just need a better explanation of what a sensor circuit is, using words like wires and sensors and then say something like we have 12 wires.

Done. I think?

**Comment by xandrew-lynx** [comment]
LGTM

**Comment by olahg** [comment]
Yes, this is much better. LGTM

**Pull request merged**